contrib/kafka/samples/README.md

```diff
@@ -117,7 +117,7 @@ and an Event Display Service.
    ```
    $ kubectl logs kafka-source-xlnhq-5544766765-dnl5s
    ...
-   {"level":"info","ts":1553034726.5351956,"logger":"fallback","caller":"adapter/adapter.go:121","msg":"Received: {value 15 0 {\"msg\": \"This is a test!\"} <nil>}"}
+   {"level":"info","ts":1554145778.9344022,"logger":"fallback","caller":"adapter/adapter.go:80","msg":"Received: {topic: 15 0 ... <nil>} {partition: 11 2  <nil>} {offset: 11 0  <nil>}"}
```

Comment:
Is this what the log messages look like? I wouldn't expect go structs to be logged here, given the log call at https://github.com/knative/eventing-sources/pull/329/files#diff-ca72cd4e88c155c7eabbb1bbd3d55b2fR80.

I'd expect something like this:

```
{"level":"info","ts":1554145778.9344022,"logger":"fallback","caller":"adapter/adapter.go:80",
"msg":"Received","topic":"mytopic","partition":11,"offset":151}
```

Reply:
Looked into this a bit now.  Yes, that's what the logs look like.  It appears that both prior logging messages and current ones are using the fallback logger (which is why we have the extra output).

Testing this quickly locally by commenting out the 
`logger := logging.FromContext(context.TODO())`
line and adding in
```
logger := zap.NewExample()
defer logger.Sync()
```
Seems to produce the logging format you describe.

May I suggest that, due to this being a preexisting issue, we leave it for another, more focused Issue/PR regarding the use of the fallback logger?

Reply:
Sure, I created #332.